opensubtitles/discover_test.go

```diff
@@ -0,0 +1,16 @@
+package opensubtitles
+
+import (
+	"testing"
+)
+
+const DiscoverServiceMostDownloadedTestData = ``
+
+
+func TestDiscoverService_MostDownloaded(t *testing.T) {
+	//TODO:
+}
+
+func TestDiscoverService_Popular(t *testing.T) {
```

Comment:
Codacy found an issue: [parameter 't' seems to be unused, consider removing or renaming it as _](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)